draft-ietf-httpbis-http2bis.xml

```diff
@@ -4353,7 +4240,7 @@
              None
```

Comment:
This section can probably be a tombstone only.  "Previous versions (see X of 7540) registered an upgrade token, but this capability has been removed; see Section X."